### ISSUE TITLE
Floating point exception (core dumped) with Conv2d 

### ISSUE THREAD

**Issue by rotabulo** [pytorch/pytorch#9465]
## Issue description

The following code produces a `Floating point exception (core dumped)` on Volta series with fp16:
```
import torch
from torch.nn import Conv2d
conv=torch.nn.Conv2d(256, 512, kernel_size=(3, 3), stride=(2, 2), padding=(1, 1), bias=False).cuda().half()
x=torch.rand(2,256,196,196, requires_grad=True).cuda().half()
y=conv(x)
loss=y.min()
loss.backward()
```

By changing the convolution parameters by +1/-1 the code works.

I suspect this to be a cudnn bug.

## System Info
```
Collecting environment information...
PyTorch version: 0.4.0a0+3749c58
Is debug build: No
CUDA used to build PyTorch: 9.2.88

OS: Ubuntu 16.04.4 LTS
GCC version: (Ubuntu 5.4.0-6ubuntu1~16.04.10) 5.4.0 20160609
CMake version: version 3.5.1

Python version: 3.5
Is CUDA available: Yes
CUDA runtime version: 9.2.88
GPU models and configuration: 
GPU 0: TITAN V
GPU 1: TITAN V
GPU 2: TITAN V
GPU 3: TITAN V

Nvidia driver version: 396.26
cuDNN version: Probably one of the following:
/usr/local/cuda-9.2/cuda/lib64/libcudnn.so.7.1.4
/usr/local/cuda-9.2/cuda/lib64/libcudnn_static.a
/usr/local/cuda-9.2/targets/x86_64-linux/lib/libcudnn.so.7.1.4
/usr/local/cuda-9.2/targets/x86_64-linux/lib/libcudnn_static.a

Versions of relevant libraries:
[pip3] numpy (1.14.5)
[pip3] torch (0.4.0a0+3749c58)
[pip3] torchvision (0.2.1)
[conda] Could not collect
```

Reproducible also on V100.

**Comment by soumith** [comment]
I couldn't reproduce this. Did you have to use a specific set of weights? maybe setting the random seed in your repro `torch.manual_seed(seed) # seed is an integer` will help me reproduce the issue.

**Comment by rotabulo** [comment]
@soumith I could reproduce irrespective of input or weights. The only variable that affect the outcome are the convolution parameters. I do not a Volta machine available right now to fix the seed, but I will do asap. If it can help, the bug is triggered only on volta series and only when a conv algorithm based on tensor cores is selected (e.g. by turning off cudnn.benchmark the code above might work).

**Comment by rotabulo** [comment]
@soumith I have tried different seeds and I always get the same exception.

**Comment by soumith** [comment]
cudnn 7.2 fixes this, so closing the issue.
